cnf-certification-test/networking/suite.go

```diff
@@ -121,8 +121,26 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
 		testPartnerSpecificTCPPorts(&env)
 	})
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestDpdkCPUPinningExecProbe)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		dpdkPods := env.GetCPUPinningPodsWithDpdk()
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, dpdkPods)
+		testExecProbDenyAtCPUPinning(dpdkPods)
+	})
 })
 
+func testExecProbDenyAtCPUPinning(dpdkPods []*provider.Pod) {
+	ginkgo.By("Check if exec probe is happening")
+
+	for _, cpuPinnedPod := range dpdkPods {
+		for _, x := range cpuPinnedPod.Containers {
+			if len(x.Command) != 0 || len(x.Args) != 0 {
```

Comment:
is this the right command to check here? This seems to be the main entry point for the container which is ok to block. Shouldn't the test be for liveliness and readiness probes that are happening in parallel?

Reply:
I am not sure why liveliness and readiness probe in parallel should be blocked. @mwlinca Is that what is required?

Reply:
exec probes confirmed this morning